idaes_examples/archive/power_gen/rsofc/rsofc_soec_src.ipynb

```diff
@@ -193,7 +193,7 @@
+       "    <text xml:space=\"preserve\" style=\"font-style:normal;font-variant:normal;font-weight:bold;font-stretch:normal;font-size:2.82222px;line-height:1.25;font-family:sans-serif;-inkscape-font-specification:'sans-serif, Bold';font-variant-ligatures:normal;font-variant-caps:normal;font-variant-numeric:normal;font-variant-east-asian:normal;text-align:end;text-anchor:end;stroke-width:0.264583\" x=\"174.74294\" y=\"139.66795\" id=\"text3524-6-5-1\"><tspan sodipodi:role=\"line\" x=\"174.74294\" y=\"139.66795\" style=\"font-style:normal;font-variant:normal;font-weight:bold;font-stretch:normal;font-size:2.82222px;font-family:sans-serif;-inkscape-font-specification:'sans-serif, Bold';font-variant-ligatures:normal;font-variant-caps:normal;font-variant-numeric:normal;font-variant-east-asian:normal;text-align:center;text-anchor:middle;stroke-width:0.264583\" id=\"tspan67246-8\">Hydrogen </tspan><tspan sodipodi:role=\"line\" x=\"174.74294\" y=\"143.19572\" style=\"font-style:normal;font-variant:normal;font-weight:bold;font-stretch:normal;font-size:2.82222px;font-family:sans-serif;-inkscape-font-specification:'sans-serif, Bold';font-variant-ligatures:normal;font-variant-caps:normal;font-variant-numeric:normal;font-variant-east-asian:normal;text-align:center;text-anchor:middle;stroke-width:0.264583\" id=\"tspan104292\">purification </tspan><tspan sodipodi:role=\"line\" x=\"174.74294\" y=\"146.72351\" style=\"font-style:normal;font-variant:normal;font-weight:bold;font-stretch:normal;font-size:2.82222px;font-family:sans-serif;-inkscape-font-specification:'sans-serif, Bold';font-variant-ligatures:normal;font-variant-caps:normal;font-variant-numeric:normal;font-variant-east-asian:normal;text-align:center;text-anchor:middle;stroke-width:0.264583\" id=\"tspan104296\">and</tspan><tspan sodipodi:role=\"line\" x=\"174.74294\" y=\"150.25128\" style=\"font-style:normal;font-variant:normal;font-weight:bold;font-stretch:normal;font-size:2.82222px;font-family:sans-serif;-inkscape-font-specification:'sans-serif, Bold';font-variant-ligatures:normal;font-variant-caps:normal;font-variant-numeric:normal;font-variant-east-asian:normal;stroke-width:0.264583\" id=\"tspan104298\"><tspan style=\"font-style:normal;font-variant:normal;font-weight:bold;font-stretch:normal;font-size:2.82222px;font-family:sans-serif;-inkscape-font-specification:'sans-serif, Bold';font-variant-ligatures:normal;font-variant-caps:normal;font-variant-numeric:normal;font-variant-east-asian:normal;text-align:center;text-anchor:middle\" id=\"tspan124110\">compressor</tspan>n</tspan></text>\n",
```

Comment:
Looking at the diagrams, the typo appears in the label "hydrogen purification and compressio" and should be corrected to "compression", not "compressor". @ksbeattie when you have a moment, could you please update this file and regenerate the SVG files as well?

Reply:
ok, this was tricky!

What was happening is `typos` was complaining about the string "compressio" in the raw xml within an .svg file (and the same within a notebook) but that same rendered svg image showed "compression" (with the 'n') both via a web page and inside of Inkscape.  I think this is just a fluke of the way Inkscape generated the xml for this image that it broke apart the last "n" in "compression" in the XML.  And I missed it since I was only editing the raw xml and wasn't even looking at the image rendered.

I think I've got this fixed now - again by editing the raw xml but this time more intelligently and looking at the resulting image files.

Please double-check!